recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
   run:
     - python >=3.9
     - numpy >=1.21
-    - pandas >=1.4,<2
+    - pandas >=1.4
```

Comment:
Another 

![](https://media4.giphy.com/media/4eEYtfuVVaEv7YrTGk/200w.gif?cid=6c09b952romx5unrkxtubbxddg683dwys19b6i92tchsbb7l&rid=200w.gif&ct=g)

for the bot inspection!